vault/resource_ssh_secret_backend_role.go

```diff
@@ -67,6 +67,11 @@ func sshSecretBackendRoleResource() *schema.Resource {
 			Type:     schema.TypeString,
 			Optional: true,
 		},
+		"allowed_domains_template": {
+			Type:     schema.TypeBool,
+			Optional: true,
+			Default:  false,
```

Comment:
Since Vault already sets a Default for this field, I suggest removing the `Default` parameter and instead making this field `Computed: true`. This means that this field can also be read-only from Vault and set to state if not provided by the user. We prefer to let Vault handle any defaults that it sets and only set default values if we really need to :)

Reply:
thanks - I've removed defaults and added computed.